guardrails/guard.py

```diff
@@ -397,7 +397,7 @@ def parse(
         full_schema_reask: bool = None,
         *args,
         **kwargs,
-    ) -> Union[Tuple[str, Dict], Awaitable[Tuple[str, Dict]]]:
+    ) -> Union[str, Awaitable[str]]:
```

Comment:
That's not quite right, it would be either a str or a Dict depending on if the output is structured or not